docs/sources/mimir/migration-guide/migrate-from-cortex.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Migrate from Cortex to Grafana Mimir"
+menuTitle: "Migrate from Cortex"
+description: "Learn how to migrate your deployment of Cortex to Grafana Mimir to simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage."
+weight: 10
+---
+
+# Migrate from Cortex to Grafana Mimir
+
+This document guides an operator through the process of migrating a Jsonnet deployment of [Cortex](https://cortexmetrics.io/) to Grafana Mimir.
+This document includes an overview of the steps required for any environment, and specific instructions for [environments deployed with Jsonnet](#migrating-to-grafana-mimir-using-jsonnet).
+This document was tested with Cortex versions 1.10 and 1.11. While it may work with more recent versions of Cortex, we cannot guarantee that.
+To migrate a Helm deployment of Cortex refer to [Migrate from Cortex](/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-cortex).
```

Comment:
```suggestion
> **Note:** This document was tested with Cortex versions 1.10 and 1.11. It might work with more recent versions of Cortex, but that is not guaranteed.
To migrate a Helm deployment of Cortex refer to [Migrate from Cortex]({{< relref "/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-cortex" >}}).
```

Reply:
Not sure this relref is going to work, but use relrefs whenever possible because they are testing during CI.

Reply:
the relref didn't work. I will revert the relref change

Reply:
Yeah, I did not mean to change the order.

Reply:
ok, i will use the previous order